### PR TITLE
Add guidance

### PR DESCRIPTION
**Pull request by johnhany97**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by johnhany97** [comment]
I accept to be a maintainer

**Comment by johnhany97** [comment]
@conda-forge/help-python please review and merge my recipe

File: recipes/guidance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    Guidance enables you to control modern language models more effectively and efficiently
+    than traditional prompting or chaining. Guidance programs allow you to interleave generation,
+    prompting, and logical control into a single continuous flow matching how the language model
+    actually processes the text. Simple output structures like Chain of Thought and its many
+    variants (e.g., ART, Auto-CoT, etc.) have been shown to improve LLM performance.
+    The advent of more powerful LLMs like GPT-4 allows for even richer structure, and guidance
+    makes that structure easier and cheaper.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipes/guidance/meta.yaml:57]: The license file is in the `sdist`. It just doesn't have the `.txt` extension. So let's update this like so to pick up the license file from the `sdist`

```suggestion
  license_file: LICENSE
```

File: recipes/guidance/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jakirkham) [inline recipes/guidance/LICENSE.txt:1]: Please drop this file since it is in the `sdist`

**Review by jakirkham — COMMENTED** [review]
Thank John! 🙏

Generally this looks good. Had a small suggestion below

**Comment by johnhany97** [comment]
All done - thank you for reviewing @jakirkham :)

**Comment by jakirkham** [comment]
Thanks John! 🙏

**Pull request merged**